packages/shared/lib/participation/stores.ts

```diff
@@ -207,6 +207,24 @@ export const shimmerStakingRewards: Readable<number> = derived(participationOver
     sumStakingRewards(StakingAirdrop.Shimmer, overview)
 )
 
+/**
+ * The total accumulated rewards for selected account
+ */
+export const selectedAccountStakingRewards: Readable<{ assembly: number; shimmer: number }> = derived(
```

Comment:
This has to depend on the selected account also, otherwise it will not work properly